lib/rubocop/cop/sorbet/multiple_t_enum_values.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Sorbet
+      # Disallow creating a `T::Enum` with less than two values.
+      #
+      # @example
+      #
+      #  # bad
+      #  class ErrorMessages < T::Enum
+      #    enums do
+      #      ServerError = new("There was a server error.")
+      #    end
+      #  end
+      #
+      #  # good
+      #  class ErrorMessages
+      #    SERVER_ERROR = "There was a server error."
+      #  end
```

Comment:
The `good` one is confusing in this case, as the cop do not really advise to use a constant instead. I think a better `good` here may be an enum with 2 values?